cfgov/unprocessed/css/on-demand/blocks/car-buying-guide.less

```diff
@@ -0,0 +1,72 @@
+@import (reference) '../../main.less';
+
+.respond-to-print({
+    .content_main {
+        border-width: 0 !important;
+        margin: 0 !important;
+        display: block !important;
+        position: static !important;
+
+        div.block {
+            display: none !important;
+        }
+
+        .block.cbg-print-block {
+            div[class^="block"] {
+                display: block !important;
+            }
+            display: block !important;
+        }
+
+        &:after {
+            border-left: none !important;
+            position:static;
+        }
+    }
+
+    header,
+    footer,
+    .content_sidebar,
+    .cbg-checklist {
+        display: none !important;
+    }
+
+    .cbg-print-paper {
+        display: block !important;
+        position: static !important;
+
+        table {
+            td + td {
+                border-left: 1px solid #cfcfcf;
+            }
+        }
+
+        thead {
+            th, td {
+                margin-bottom: unit( 14px / @base-font-size-px, em );
+                font-weight: 600;
+                letter-spacing: 1px;
+                line-height: 1.25;
+                text-transform: uppercase;
+                padding: unit( 12px / @base-font-size-px, em );
+                color: @black;
+                background: transparent;
+            }
+        }
+
+        tbody {
+            tr {
+                border-bottom: 1px solid #cfcfcf;
+            }
+
+            th, td {
+                padding-bottom: 1.2em;
+                padding-top: 1.2em;
+            }
+
+            td + td {
+                padding-left: 1.2em;
```

Comment:
Ems should probably use the unit() utility used elsewhere in this file.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5780/commits/843b349e9a8ba91c1f5cebd2c165ea558305f751